src/trackers/twitter.ts

```diff
@@ -0,0 +1,50 @@
+import {loadTwitterTag, loadTwitterUniversal} from "../utils/externalServices";
+import {BaseTracker, PageMeta} from "./base";
+
+declare var twq: any;
+declare var twttr: any;
+
+
+export interface TwitterOptions {
+  mainTid: string;
+  booksRegisterTid?: string;
+  selectRegisterTid?: string;
+  productTrackingTid: string;
```

Comment:
트위터의 경우 pixel id가 각 이벤트 별로 설정되어 있는 걸로 추정되는데요,
ex) `booksRegisterTid`의 경우 리디북스 회원가입 완료 이벤트 전송 pixel Id를 의미
전역설정에 쓰이는 `mainTid` 외의 나머지는 실제 관련된 이벤트 함수명이 반영된 이름을 사용하면 어떨까 합니다. ex) `registrationPid`
@ebongzzang 님 의견은 어떠신가요?

Reply:
https://github.com/ridi/event-tracker/pull/33/files#r454145141 에서 제안드린대로 수정하면 

```suggestion
  mainTid: string;
  booksSignUpTid?: string;
  selectRegistrationTid?: string;
  ImpressionTid: string;
```

이렇게 될 것 같습니다.